ark/util/serialize.ts

```diff
@@ -102,6 +105,14 @@ const _serialize = (
 			const result: Record<string, unknown> = {}
 			for (const k in o) result[k] = _serialize((o as any)[k], opts, nextSeen)
 
+			for (const s of Object.getOwnPropertySymbols(o)) {
```

Comment:
`getOwnPropertySymbols` is needed because `Object.keys` and `Object.entries` or the `in` operator above do not include symbol keys. This just addresses the symbol keys on their own.